src/react/databrowser/buckets/__tests__/BucketCreate.test.jsx

```diff
@@ -0,0 +1,107 @@
+import * as actions from '../../../actions';
+import BucketCreate from '../BucketCreate';
+import React from 'react';
+import { act } from 'react-dom/test-utils';
+import { reduxMount } from '../../../utils/test';
+
+// Jest requires a setter on the tested object but ES6 doesn't have
+// but jest.mock() allows you solving this by mocking your required module after the import.
+jest.mock('../../../actions', () => ({
```

Comment:
Ok so I did some research and  I think it is not possible to mock clearServerError method from the component for many reasons:
1) the fact thar CreateBucket is not a class, indeed in this issue https://github.com/enzymejs/enzyme/issues/944 someone manage to spyOn a function inside a component using instance() but his component is a class and according to the enzyme documentation we can't do an instance() on a functional component since react 16.x , you can check it here: https://enzymejs.github.io/enzyme/docs/api/ShallowWrapper/instance.html
2) the other way was to use dive() instead of instance() but this needs to use shallow instead of mount which is not possible as BucketCreate need the store to work but shallow doesn't render the entire tree only the first children